inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsIdConfig.java

```diff
@@ -46,10 +46,12 @@
      */
     public List<String> getFieldList() {
         if (fieldList == null) {
-            this.fieldList = new ArrayList<>();
-            if (fieldNames != null) {
-                String[] fieldNameArray = fieldNames.split("\\s+");
-                this.fieldList.addAll(Arrays.asList(fieldNameArray));
+            synchronized (fieldNames) {
```

Comment:
This line will throw Exception when fieldNames==null.
It is better to invoke this method in reload method.

Reply:
thx, fixed it